2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: invalid input syntax for type integer: "h"
LINE 4: ... timeframe = '1'                       AND sequence_id = 'h'
                                                                    ^

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 15:59:39 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-24 15:59:39 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: invalid input syntax for type integer: "h"
LINE 4: ... timeframe = '1'                       AND sequence_id = 'h'
                                                                    ^

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:18:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-24 16:18:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: invalid input syntax for type integer: "h"
LINE 4: ... timeframe = '1'                       AND sequence_id = 'h'
                                                                    ^

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:26:02 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-24 16:26:02 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: current transaction is aborted, commands ignored until end of transaction block

2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: DatabaseManager.get_btc_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: DatabaseManager.get_btc_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: DatabaseManager.get_btc_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: DatabaseManager.get_btc_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: DatabaseManager.get_eth_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: DatabaseManager.get_eth_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: DatabaseManager.get_eth_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-24 16:28:51 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: DatabaseManager.get_eth_lstm_sequence() got an unexpected keyword argument 'timeframe_value'
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: Не вдалося створити жодних ознак для BTC
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: Не вдалося створити жодних ознак для BTC
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: Не вдалося створити жодних ознак для BTC
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: Не вдалося створити жодних ознак для BTC
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: Не вдалося створити жодних ознак для ETH
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: Не вдалося створити жодних ознак для ETH
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: Не вдалося створити жодних ознак для ETH
2025-05-24 16:52:58 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-24 16:52:58 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: Не вдалося створити жодних ознак для ETH
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-25 01:56:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-25 02:01:01 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-25 02:07:13 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: 'NoneType' object is not subscriptable
2025-05-25 02:17:26 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 02:17:27 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: list indices must be integers or slices, not str
2025-05-25 02:17:27 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-25 02:17:29 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: list indices must be integers or slices, not str
2025-05-25 02:17:29 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-25 02:17:29 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: list indices must be integers or slices, not str
2025-05-25 02:17:29 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-25 02:17:30 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: list indices must be integers or slices, not str
2025-05-25 02:17:30 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-25 02:17:31 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: list indices must be integers or slices, not str
2025-05-25 02:17:31 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-25 02:17:33 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: list indices must be integers or slices, not str
2025-05-25 02:17:33 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-25 02:17:33 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: list indices must be integers or slices, not str
2025-05-25 02:17:33 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-25 02:17:34 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: list indices must be integers or slices, not str
2025-05-25 02:24:35 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 02:24:36 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: list indices must be integers or slices, not str
2025-05-25 02:24:36 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-25 02:24:38 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: list indices must be integers or slices, not str
2025-05-25 02:24:38 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-25 02:24:38 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: list indices must be integers or slices, not str
2025-05-25 02:24:38 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-25 02:24:39 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: list indices must be integers or slices, not str
2025-05-25 02:24:39 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-25 02:34:06 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 02:34:08 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-lstm: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:08 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_gru
2025-05-25 02:34:09 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1h-gru: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:09 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-25 02:34:10 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:10 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-25 02:34:10 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:10 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_lstm
2025-05-25 02:34:12 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-lstm: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:12 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1h_gru
2025-05-25 02:34:13 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1h-gru: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:13 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-25 02:34:14 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:34:14 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-25 02:34:14 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: Очікується DataFrame, отримано <class 'list'>
2025-05-25 02:38:11 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-25 02:43:18 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-26 23:56:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:00:25 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:07:16 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:15:39 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:28:31 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 01:21:48 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 01:28:48 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1d_lstm
2025-05-27 01:32:57 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 01:33:11 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 01:33:11 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 01:33:25 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 01:33:25 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 01:33:40 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 01:33:40 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 01:33:54 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 01:33:54 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 01:34:09 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 01:34:09 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 01:34:23 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-transformer: "['target'] not found in axis"
2025-05-27 01:34:23 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_lstm
2025-05-27 01:34:37 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-lstm: "['target'] not found in axis"
2025-05-27 01:34:37 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_gru
2025-05-27 01:34:51 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-gru: "['target'] not found in axis"
2025-05-27 01:34:51 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_transformer
2025-05-27 01:35:05 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-transformer: "['target'] not found in axis"
2025-05-27 12:40:22 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 12:40:37 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 12:40:37 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 12:40:51 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 12:40:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
